CuBoulder/express
# removing osr test and staff users on cron #2586

#2586 

To test:

1. Make a new site
2. Enable the `dblog` module 
3. Login with osr test accounts to populate `admin/people`
4. Login with your own identikey credentials, sign out
5. Go to `admin/config/system/cron` and force run `cu_users_cron
6. Got to `admin/reports/dblog` and filter by `cu_inactive_users`
7. Check to for the log that looks like `cu_inactive_users | 07/10/2018 - 09:11 | Deleted users: administrator, developer | Jesus Gabriel Ortiz Tovar (Web Express Team)`
